### DISCUSSION TITLE
Gerador de ameaça

### DISCUSSION THREAD

**Discussion by LeandroMohr** [YuriAlessandro/gerador-ficha-tormenta20#342] [Ideias]
Salve na paz?

Estou achando muito útil a ferramenta mas infelizmente o link para geração de ameaças/monstros está quebrando 
![Erro 503](https://user-images.githubusercontent.com/30901466/215266372-313f9468-9fab-479d-bcf7-a381541fc384.png).

Na minha visão essa seria uma ótima implementação.

**Comment by betagame13** [comment]
Eu também gostei, infelizmente não está pegando, ha só para avisar esta na categoria errada.

**Comment by YuriAlessandro** [comment]
Fala rapaziada desculpa a demora em responder. O Gerador de Ameaças que está linkado no site é um projeto de terceiros, que o dono criou. Eu estava criando um interno para o Fichas de Nimb mas achei que o dele já estava funcionando legal e resolvi apenas linkar. 

Vou tentar entrar em contato com ele para ser corrigido. Se não, vou apenas remover o link - e provavelmente começar a trabalhar num próprio gerador de ameaças, pra não correr esse risco
